sdk/parents/clientcore-parent/pom.xml

```diff
@@ -1517,5 +1487,47 @@
         <spotbugs.excludeFilterFile>spotbugs-exclude.xml</spotbugs.excludeFilterFile>
       </properties>
     </profile>
+
+    <profile>
+      <id>add-changelog-and-readme</id>
+      <activation>
+        <file>
+          <exists>src</exists>
+        </file>
+      </activation>
```

Comment:
was this the only change, why not add it above?

Reply:
Only `profile` supports activation configuration.